pkg/validation/spec/info.go

```diff
@@ -53,11 +53,16 @@ func (e Extensions) GetBool(key string) (bool, bool) {
 // GetStringSlice gets a string value from the extensions
 func (e Extensions) GetStringSlice(key string) ([]string, bool) {
 	if v, ok := e[strings.ToLower(key)]; ok {
+		var strs []string
+		if stringSlice, isStringSlice := v.([]string); isStringSlice {
+			strs = make([]string, len(stringSlice))
+			copy(strs, stringSlice)
```

Comment:
I guess the copy is good since that's what happens for the other case. Not sure how much that's needed but good.